src/pages/_proxied-dot-io/packer/community/index.jsx

```diff
@@ -0,0 +1,49 @@
+import VerticalTextBlockList from '@hashicorp/react-vertical-text-block-list'
+import SectionHeader from '@hashicorp/react-section-header'
+import Head from 'next/head'
+import s from './style.module.css'
+
+export default function CommunityPage() {
```

Comment:
We've gotta add the `PackerIoLayout` here 👀

https://dev-portal-git-zsadd-packer-hashicorp.vercel.app/community